Oswaldo Vazquez Aparicio/30 Agosto/hackerrank/JavaList.java

```diff
@@ -2,11 +2,12 @@
 
 import java.util.LinkedList;
 import java.util.Scanner;
+import java.util.List;
 
 public class JavaList {
 
     private static final Scanner scanner = new Scanner(System.in);
-    private static final LinkedList<Integer> list = new LinkedList<>();
+    private static final List<Integer> list = new LinkedList<>();
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableListList, userSet, etc...